docker/orchestration/run_services.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+service nginx start
```

Comment:
Out of curiosity, what's the precondition to run `nginx` when not using Docker? Do we need any extra edit on here?

https://github.com/google/android-cuttlefish/blob/main/README.md#debian-packages

Reply:
While on chat, doing `apt install` for existing 3 CF debian packages would be okay. Before that, `apt install nginx` seems to be needed.

Reply:
when installing the packages directly in the host, `systemd` starts the service, there's no need to start it manually as within a container.